dipy/workflows/stats.py

```diff
@@ -539,14 +538,14 @@ def run(self, subject_folder, clust_thr=[5, 3, 1.5], threshold=6,
             Path to the input subject folder. This path may contain
             wildcards to process multiple inputs at once.
 
-        clust_thr : variable float (default [5,3,1.5]), optional
-            list of bundle clustering thresholds used in quickbundlesX
+        clust_thr : variable float, optional
+            list of bundle clustering thresholds used in quickbundlesX.
```

Comment:
Since the line was modified, can we change the case and write "QuickBundlesX", please?

Reply:
Addressed in e5af17f